UKHomeOffice/cerberus-service
# COP-5041 Task summary

## Description
Add the task summary section to the task details page

- added task summary component
- added a formatTaskData util
- refactored so the summary on the list view and the summary on the details view use the same formatting rules (formatTaskData util)
- removed the temporary data from the task details section as it was using summary data incorrectly


## To Test

- open task list page
- > layout and titles should match prototype
- open task details page
- > layout and titles should match prototype (note, sample tasks do not have trailer so this is not shown in the summary text, but is shown on the prototype)


## Developer Checklist

\* Required
